docs/source/notes/cuda.rst

```diff
@@ -255,12 +259,6 @@ have the same stream-semantics relationship as any group of ops::
         initial_grad.record_stream(s)
         loss.backward(gradient=initial_grad)
 
-If your forward pass runs some independent ops in parallel on different streams,
```

Comment:
What about adding a sentence or two here that in prior versions of PyTorch (1.9 and below) these operations synced with the default CUDA stream, but this is no longer the case. Then you could suggest how people could restore the old behavior by calling `torch.cuda.synchronize()` if they were concerned?

Reply:
Added https://github.com/pytorch/pytorch/pull/60421/files#diff-4840823ff4b0cd7fe61b9ada2a1746f6936c944bfbcc4d1bdb8b3dc1561d4401R262-R281